packages/lisk-transactions/src/1_second_signature_transaction.ts

```diff
@@ -209,6 +210,8 @@ export class SecondSignatureTransaction extends BaseTransaction {
 	protected undoAsset(store: StateStore): ReadonlyArray<TransactionError> {
 		const sender = store.account.get(this.senderId);
 		const { secondPublicKey, ...strippedSender } = sender;
```

Comment:
```suggestion
		const { secondPublicKey, secondSignature, ...strippedSender } = sender;
```

Reply:
@shuse2 I think we need the properties to be present for stateStore to handle them correctly so instead changed to de-structuring as we discussed.